include/dpp/timed_listener.h

```diff
@@ -73,7 +73,7 @@ template <typename attached_event, class listening_function> class timed_listene
 		/* Attach event */
 		listener_handle = ev(listener);
 		/* Create timer */
-		th = cl->start_timer([this](dpp::timer timer_handle) {
+		th = cl->start_timer([this]([[maybe_unused]] dpp::timer timer_handle) {
```

Comment:
Why is this "maybe unused"? When would you use a timer without a lambda or storing the timer 🤔